public/index.ts

```diff
@@ -1082,13 +1111,74 @@ function createTestVolume() {
   };
 }
 
+function createLoader(type: string): IVolumeLoader {
```

Comment:
this function could go into IVolumeLoader and become the primary (or only?) way to create a IVolumeLoader.

Reply:
I like this as an option for apps that may switch between many file formats, but I also like the idea of clients that don't expect to use more than one format only needing to import the loader they need.

Reply:
Yeah, probably most useful as a way to pass in a LoadSpec, and have some logic to automatically figure out what kind of loader to use so that the client doesn't need to care.  Factory method pattern.  But not sure how often that will come up.  Something for down the road...